### PR TITLE
upgrade packages

### PR DESCRIPTION
**Pull request by lloydbenson**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,11 +8,11 @@
         "lib"
     ],
     "dependencies": {
-        "@hapi/pinpoint": "1.x.x"
+        "@hapi/pinpoint": "2.x.x"
```
Comment (cjihrig) [inline package.json:11]: This module has already moved out of the hapi org, but a new version hasn't been published yet. See https://github.com/sideway/pinpoint

Comment (lloydbenson) [reply package.json:11]: Thx for letting me know! You think we should just wait on this one then?

Comment (cjihrig) [reply package.json:11]: @hueniverse are you planning to publish a new pinpoint any time soon?

Comment (hueniverse) [reply package.json:11]: Not soon. This is a test module so you can just leave as-is.

**Review by cjihrig — APPROVED** [review]

**Pull request merged**